lib/gauge_stout.cu

```diff
@@ -26,14 +27,15 @@ namespace quda {
 
   public:
     // (2,3/4): 2 for parity in the y thread dim, 3 or 4 corresponds to mapping direction to the z thread dim
-    GaugeSTOUT(GaugeField &out, const GaugeField &in, bool improved, double rho, double epsilon, int dir_ignore) :
+    GaugeSTOUT(GaugeField &out, const GaugeField &in, bool improved, double rho, double epsilon, int dir_ignore, const double anisotropy) :
```

Comment:
There's no need to make a pass-by-value variable `const` (I know a bunch of code does this already in QUDA, but we're working towards fixing this).